tests/test_openapi.py

```diff
@@ -459,13 +443,110 @@ def test_cql_filters_capabilities(get_cql_schemas, is_cql):
         filter_capabilities = ['conformance-classes',
                                'capabilities', 'functions']
         for props in filter_capabilities:
-            assert props in get_cql_schemas['filter-capabilities']['properties'] # noqa
+            assert props in get_cql_schemas['filter-capabilities']['properties']  # noqa
+
+
+def test_cql_queryables_path(get_oas_30_, get_collections, is_cql):
```

Comment:
@jorgejesus @francbartoli queryables I have made separate test functions to deal with assertion of root and local queryables